tests/test_lib_tokens_push.py

```diff
@@ -102,9 +102,9 @@ def test_01_create_token(self):
                       "pubkey": self.smartphone_public_key_pem})
         self.assertEqual(token.get_tokeninfo("firebase_token"), "firebasetoken")
         self.assertEqual(token.get_tokeninfo("public_key_smartphone"), self.smartphone_public_key_pem)
```

Comment:
For Python 3 compatibility, I think we either need to convert ``token.get_tokeninfo("public_key_smartphone")`` (which is a unicode string) to a bytestring, or ``self.smartphone_public_key_pem`` (a bytestring) to unicode.

Reply:
done in 9e0e477d